src/components/TableRow.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <tr @click="$emit('rowClick', row)">
+    <tr @click="$emit('row-click', row)">
```

Comment:
Changing this will break backwards compatibility.

Could you fix this by emitting both `row-click` and `rowClick`?